### DISCUSSION TITLE
are coding questions a little bit hard for beginners

### DISCUSSION THREAD

**Discussion by s1ddiq** [codewars/codewars.com#3246] [Polls]
just wondering since imn a darn beginenrer

**Comment by s1ddiq** [comment]
now that im thinking about what i just said i look stupid

**Reply by AbyTed** [reply]
we all been there.

**Reply by s1ddiq** [reply]
lolelrs

**Reply by binarybadg3r** [reply]
Nah

**Reply by jocypher** [reply]
nope it is normal.

**Comment by LucasCozzer** [comment]
At first I thought I was useless and that programming really wasn't for me, then I discovered that I skipped the most important part... logic!

**Reply by AbyTed** [reply]
Skill issue

**Reply by s1ddiq** [reply]
lol

**Reply by s1ddiq** [reply]
> At first I thought I was useless and that programming really wasn't for me, then I discovered that I skipped the most important part... logic!

i agree actually

**Reply by ogeobubu** [reply]
For Backend Engineer right? Then agree. I doubt if that is required for a Frontend Engineer; one just generally needs common sense to be a programmer.

**Reply by binarybadg3r** [reply]
Logic is everything!

**Reply by jocypher** [reply]
Yeah logic is indeed an important part , however I believe understanding what you are supposed to do makes it better

**Reply by jocypher** [reply]
> For Backend Engineer right? Then agree. I doubt if that is required for a Frontend Engineer; one just generally needs common sense to be a programmer.

nope not only backend engineer but every engineer.

**Comment by aelishkumar8** [comment]
Everyone was a beginner at first, but learning and practicing will make them better every time. So, for me, I love to play online games (FPS) and also I love to learn coding. Starting my coding journey was a little hard but I tried with small steps and now I am better in coding. So, we all are the same 😄

**Comment by YeshanSheranga** [comment]
Start coding basic algorithms problems and end of the day anyone will not be able to get progress but after one to two months skills level
will increase. always try measure the skills level then its suitable i think
